cukedoctor-multipage-layout/src/main/java/cukedoctor/multipage/api/MultipageConverter.java

```diff
@@ -0,0 +1,74 @@
+package cukedoctor.multipage.api;
+
+import com.github.cukedoctor.Cukedoctor;
+import com.github.cukedoctor.api.CukedoctorConverter;
+import com.github.cukedoctor.util.FileUtil;
+import com.github.cukedoctor.util.ServiceLoaderUtil;
+import cukedoctor.multipage.defaults.DefaultMultipagePersister;
+import cukedoctor.multipage.defaults.OnePagePerFeatureMultipagePager;
+import cukedoctor.multipage.model.Page;
+import cukedoctor.multipage.model.MultipageDocumentation;
+import cukedoctor.multipage.spi.MultipagePersister;
+import cukedoctor.multipage.spi.MultipagePager;
+
+import java.util.List;
+
+/**
+ * This class serves as a replacement to the {@link com.github.cukedoctor.converter.CukedoctorConverterImpl} and can
+ * handle multipage documentation.
+ *
+ * Main usage is to create a documentation sites with tools like antora.
+ */
+public class MultipageConverter {
+    private MultipagePager pager;
+    private MultipagePersister persister;
+
+    private String jsonFilesLocation = "target/test-classes/json-output/";
+    private String outputFolderLocation = "docs/modules/living/pages/";
+
+    public MultipageConverter() {
+        loadDependencies();
+    }
+
+    private void loadDependencies() {
+        this.pager = new ServiceLoaderUtil<MultipagePager>().initialise(MultipagePager.class, OnePagePerFeatureMultipagePager.class, null, null, null);
```

Comment:
It is important to specify the _CukedoctorConfig_ here so that child objects have it and respect it e.g. "hideScenarioKeyword" for the _ScenarioRenderer_ instance that must eventually be instantiated.